doc/requirements/requirements.md

```diff
@@ -0,0 +1,306 @@
+DEV004 | Tools.Static Analysis | 1 | *TODO: What is the static analysis tool to use?*
+DEV005 | Tools.Static Analysis.Quality Targets | 1 | *TODO: What are the static analysis targets?*
+DEV006 | Tools.Dynamic Analysis | 1 | *TODO: What dynamic analysis tool(s) to use?*
+DEV007 | Tools.Dynamic Analysis.Code Coverage Targets | 1 | *TODO: What are the code coverage targets?*
+DEV008 | Tools.Dynamic Analysis.Memory Checking | 1 | *TODO: What is/are the memory checking tool(s) to use?*
+
+## 2.3 Target Platforms
+
+The Navigation System will run on the latest versions of the operating systems supported by the core ROS 2 code.
+
+Id | Handle | Priority | Description | Notes
+-- | ------ | -------- | ----------- | -----
+PLT001 | Target Platforms.Operating Systems.Ubuntu | 1 | The Navigation System MUST support Ubuntu Desktop 16.04 and Ubuntu Desktop 18.04
+PLT002 | Target Platforms.Operating Systems.MacOS | 1 | The Navigation System MUST support MacOS 10.13 (High Sierra) and MacOS 10.14 (Mohave)
+PLT003 | Target Platforms.Operating Systems.Windows | 1 | The Navigation System MUST support Windows 10 Professional
+PLT004 | Target Platforms.Operating Systems.Clear Linux | 1 | The Navigation System SHOULD support the latest version of Intel's Clear Linux distribution | What is the latest version number?
+PLT005 | Target Platforms.CPU.Word Size | 1 | The Navigation System SHALL support 64-bit processors | Don't assume a specific pointer size
+PLT006 | Target Platforms.Minimum Platform | 1 | *Should we specify a minimum target platform?* *Or, should this be expressed as minimum platform requirements?*
+
+## 2.4 Command Chain Modules
+
+This section lists the requirements for the core command chain modules in the Navigation System.
+
+### 2.4.1 Mission Planning
+
+A complete system, which incorporates the Navigation System, would also have a Mission Planning subsystem that generates a Mission Plan from input provided by the User. The Mission Plan is a sequence of primitive operations and any associated policy about how the plan should be carried out. The design and implementation of the subsystem that creates the Mission Plan is outside the scope of the Navigation System, but its output, the Mission Plan, has a direct relationship to the capabilities that the Navigation System must provide. 
+
+Id | Handle | Priority | Description | Notes
+-- | ------ | -------- | ----------- | -----
+MP001 | Mission Planning.Primitives | 1 | The Mission Plan MUST be able to express the plan as a sequence of the following Primitives (MP002 - MP010)
+MP002 | Mission Planning.Primitives.Navigate to Position | 1 | The Navigation System MUST be able to navigate the Robot from its current location to a specified destination pose.
+MP003 | Mission Planning.Primitives.Navigate to Area | 1 | The Navigation System MUST be able to navigate the Robot from its current location to a specified area. | *TODO: How to define "area"?*
```

Comment:
Area may be defined as tolerance to a goal (ie. within +/- 1 meter distance). I believe this is the way it works in the ROS Navigation today